.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,3 +13,4 @@ Resolves #
 - [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
 - [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.
 - [ ] `make vendor` does not cause changes.
+- [ ] Comments have been added or updated based on the standards set in [CONTRIBUTING.md](../blob/master/CONTRIBUTING.md#comments)
```

Comment:
This link doesn't work for me. I think you just want (/CONTRIBUTING.md#comments)

Reply:
@travisn The link should work fine in a created issue, see his comment https://github.com/rook/rook/pull/1980#issuecomment-410758134

Reply:
ok, i see now it works in an issue thanks

Reply:
@jbw976 I guess the positive here is that the link is shorter this way ?